requirements-dev.txt

```diff
@@ -4,9 +4,9 @@ Appium-Python-Client<=0.52
 flake8==3.8.3
 nose==1.3.7
 splinter>=0.13.0
--e git+https://github.com/qld-gov-au/ckanext-datarequests@2.1.1-qgov#egg=ckanext-datarequests
--e git+https://github.com/qld-gov-au/ckanext-data-qld@3.0.1#egg=ckanext-data_qld
--e git+https://github.com/qld-gov-au/ckan-ex-qgov@3.1.0#egg=ckan-ex-qgov
+-e git+https://github.com/qld-gov-au/ckanext-datarequests@develop#egg=ckanext-datarequests
```

Comment:
so we are back to not needing to lock these again?

Reply:
Good point, we did change that to fixed versions deliberately. I'll change back but update them.